CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## [0.2.1]
 
 * `Exporter` class exposes now `frames`.
```

Comment:
Can you make a new version of it? It looks like it should be bumped to 0.2.2.

Reply:
@TimHoogstrate 0.2.1 was never released. Is there any reason it should be skipped?